test/fixtures/dialects/tsql/select.yml

```diff
@@ -47,6 +47,15 @@ file:
               - expression:
                   literal: "'True'"
               - keyword: WHEN
+              - expression:
+                - literal: '1'
+                - comparison_operator: '>'
+                - comparison_operator: '='
```

Comment:
can see that these are parsed as two separate identifiers rather than a single `>=`. Might be worth adding some test cases to any rules that could potentially rely on logic that looks at the comparison operator (thinking L043).

Reply:
L007 is also one to check for this

Reply:
Also maybe L006 and L049

Reply:
@jpy-git , @barrywhart I wonder if such an issue was created (to add a rule to fix such cases with "> =") as we faced with same trouble in our repo?

Reply:
@tkachenkomaria244 no rule has been created yet

Reply:
@jpy-git Josef, I've submitted an issue then: [#1997](https://github.com/sqlfluff/sqlfluff/issues/1997)

Reply:
perfect thanks!